nixos/modules/services/misc/dump1090-fa.nix

```diff
@@ -0,0 +1,157 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}:
+let
+  cfg = config.services.dump1090-fa;
+  inherit (lib) mkOption types;
+in
+{
+  options.services.dump1090-fa = {
+    enable = lib.mkEnableOption "dump1090-fa";
+
+    package = lib.mkPackageOption pkgs "dump1090-fa" { };
+
+    extraArgs = mkOption {
+      type = types.listOf types.str;
+      default = [ ];
+      description = "Additional passed arguments";
+    };
+
+    nginx = {
+      addVirtualHost =
+        lib.mkEnableOption "nginx virtual host for dump1090-fa exposing the web interface"
+        // {
+          default = true;
+        };
+      virtualHostName = lib.mkOption {
+        type = types.str;
+        default = "dump1090-fa";
+        description = "Name of the virtual host";
+      };
+    };
+  };
+
+  config = lib.mkIf cfg.enable {
+    systemd.services.dump1090-fa = {
+      description = "dump1090 ADS-B receiver (FlightAware customization)";
+      after = [ "network.target" ];
+      wantedBy = [ "multi-user.target" ];
+
+      serviceConfig = {
+        ExecStart = lib.escapeShellArgs (
+          [
+            (lib.getExe cfg.package)
+            "--net"
+            "--write-json"
+            "%t/dump1090-fa"
```

Comment:
Is using the RuntimeDirectory the right call? These files will be automatically removed whenever the service is stopped/restarted. I'd probably put it in StateDirectory= instead, and suggest how tmpfiles can be used to remove older files if desired.

Reply:
I don't have strong opinions here but:
- those files are temporary however, I don't know exactly if the entries in the produced json get invalidated or directly removed after a while that the receiver is not receiving them anymore
- it was taken from the [upstream module](https://github.com/flightaware/dump1090/blob/32235b28a0c7d25379650ecf159eb14d26d65be4/debian/dump1090-fa.service#L13)

Honestly I would leave it as it is but I refer to your opinion.